### DISCUSSION TITLE
v0.44.0-20230103.174333

### DISCUSSION THREAD

**Discussion by qvacua** [qvacua/vimr#995] [General]
* Neovim 0.8.2 😀
* Reduce app size (~14MB) by using `strip` on VimR binary.
* Dependencies updates
  - Kitura/BlueSocket@2.0.4
  - sparkle-project/Sparkle@2.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/qvacua/vimr/releases/tag/v0.44.0-20230103.174333'>v0.44.0-20230103.174333</a>.</em>

**Comment by drbrain** [comment]
I have experienced problems with keyboard responsiveness and accuracy with v0.44.0, v0.43.0 works correctly.

When typing into the `:` menu or telescope with v0.44.0 I experience a slow response compared to v0.43.0, and keystrokes are slow to show up unless I reduce my typing rate significantly.  When I type at my typical rate I see the wrong keys be repeated to fill in for the ones I actually typed.

For example, if I find a file in [telescope](https://github.com/nvim-telescope/telescope.nvim) and type "stats" it might appear in the telescope find window as "staas" where the second "t" was replaced with an "a".  Similar behavior occurs with using `/` or `:`.

I can re-install v0.44.0 record a video if you would like.

This is [my vimrc](/drbrain/vimrc/) (I realize for `:` and `/` I have two completions enabled that fight each other that I haven't fixed, but this was not a problem in v0.43.0, and the behavior occurs for telescope which doesn't have fighting completions)
